python_scripts/ensemble.py

```diff
@@ -273,7 +277,7 @@ def bootstrap_sample(x, y):
 )
 
 # %% [markdown]
-# Theoretically, 63.2% of the original data points of the original dataset will
+# Empirically, 63.2% of the original data points of the original dataset will
 # be present in the bootstrap sample. The other 36.8% are just repeated
```

Comment:
I unsure about this. 63.2% of the original dataset will be in the bootstrap sample. 36.8% of the original dataset will not be in the bootstrap sample. I don't think it is correct to say that 36.8% are repeated samples but maybe I am missing something ...

Reply:
It might be better to mention that only 63.2% of the samples are unique.